website/static/js/passwordForms.js

```diff
@@ -92,6 +92,31 @@ var BaseViewModel = oop.extend(ChangeMessageMixin, {
 var ChangePasswordViewModel = oop.extend(BaseViewModel, {
     constructor: function () {
         var self = this;
+
+        // Call constructor at the beginning so that self.password exists
+        self.super.constructor.call(this);
```

Comment:
How about adding `password: self.password` to the return value of getValidated fields, then manually assigning `self.validatedFields` and `self.validatedObservables` in the constructor.

```js
self.validatedObservables = self.getValidatedFields();
self.validatedFields = ko.validatedObservable(self.validatedObservables);
```

By the way, you'll probably want to comment the above lines; it's a bit of a clumsy workaround.